pywinauto/unittests/test_uia_element_info.py

```diff
@@ -105,6 +107,64 @@ def test_descendants_generator(self):
             descendants = [desc for desc in self.ctrl.iter_descendants(depth=3)]
             self.assertSequenceEqual(self.ctrl.descendants(depth=3), descendants)
 
+
+    class UIAElementInfoRawViewWalkerTests(UIAElementInfoTests):
+
+        """Unit tests for the UIAElementInfo class with enabled RawViewWalker implementation"""
+
+        def setUp(self):
+            self.assertEqual(UIAElementInfo.use_raw_view_walker, False)
+            UIAElementInfo.use_raw_view_walker = True
```

Comment:
I'd suggest to keep previous value, then get back to previous value in `tearDown`. In this case we can easily switch default value to anything without touching unit tests.

Reply:
Changed it.